client/components/seo/settings-upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import QueryPlans from 'components/data/query-plans';
+import Gridicon from 'components/gridicon';
+import PlanCompareCard from 'my-sites/plan-compare-card';
+import PlanCompareCardItem from 'my-sites/plan-compare-card/item';
+import TrackComponentView from 'lib/analytics/track-component-view';
+import formatCurrency from 'lib/format-currency';
+import { preventWidows } from 'lib/formatting';
+import { getFeatureTitle } from 'lib/plans';
+import { getPlanBySlug } from 'state/plans/selectors';
+import {
+	PLAN_BUSINESS,
+	FEATURE_ADVANCED_DESIGN,
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NO_ADS,
+	FEATURE_UNLIMITED_PREMIUM_THEMES,
+	FEATURE_UNLIMITED_STORAGE,
+	FEATURE_VIDEO_UPLOADS
+} from 'lib/plans/constants';
+
+const featuresToShow = [
+	FEATURE_UNLIMITED_STORAGE,
+	FEATURE_UNLIMITED_PREMIUM_THEMES,
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NO_ADS,
+	FEATURE_ADVANCED_DESIGN,
+	FEATURE_VIDEO_UPLOADS
+];
+
+const SeoSettingsNudge = ( { translate, plan = {}, upgradeToBusiness = noop } ) => {
+	const price = formatCurrency( plan.raw_price / 12, plan.currency_code );
+
+	return (
+		<Card className="settings-upgrade-nudge">
+			<QueryPlans />
+			<TrackComponentView eventName="calypso_seo_settings_upgrade_nudge_impression" />
+			<div className="settings-upgrade-nudge__business-plan-card">
+				<PlanCompareCard
+					title={ plan.product_name_short }
+					line={ translate( '%(price)s per month, billed yearly', { args: { price } } ) }
+					buttonName={ translate( 'Upgrade' ) }
+					onClick={ upgradeToBusiness }
+					currentPlan={ false }
+					popularRibbon={ true } >
```

Comment:
not a biggie, but these are places that lead me to prefer un-indenting the `>`, because the children components flow right along from the parent's attributes